README.md

```diff
@@ -280,12 +219,12 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
+
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job in implementing the changes requested by the Previous Reviewer, however, according to the previous reviewer, you are suppose to add `LICENSE` file in your project directory because it is showing `404 page not found` when someone click the `MIT` license section.  
- Below is the screenshot when someone clicks it. :arrow_down: 

![Student License](https://github.com/RolandAmini/hello--word/assets/43528594/6a5397a2-6e09-4585-9a61-0565de7cf437)